src/main/java/com/arcbees/analytics/server/ServerOptionsCallbackProvider.java

```diff
@@ -100,33 +100,34 @@ public ServerOptionsCallback get() {
         result.putText("v", ANALYTICS_VERSION + "");
         result.putText("tid", userAccount);
         result.putText("uip", getClientIpAddr(requestProvider.get()));
-        final String[] split = ((String) requestProvider.get().getAttribute(COOKIE_VALUE_KEY)).split("\\.");
+        final String[] split = ((String) requestProvider.get().getAttribute(COOKIE_VALUE_KEY))
+                .split("\\.");
         result.putText("cid", split[2] + "." + split[3]);
         return result;
     }
-    
-    private static String getClientIpAddr(HttpServletRequest request) {  
-        String ip = request.getHeader("X-Forwarded-For");  
-        if (ip == null || ip.length() == 0 || "unknown".equalsIgnoreCase(ip)) {  
-            ip = request.getHeader("Proxy-Client-IP");  
-        }  
-        if (ip == null || ip.length() == 0 || "unknown".equalsIgnoreCase(ip)) {  
-            ip = request.getHeader("WL-Proxy-Client-IP");  
-        }  
-        if (ip == null || ip.length() == 0 || "unknown".equalsIgnoreCase(ip)) {  
-            ip = request.getHeader("HTTP_CLIENT_IP");  
-        }  
-        if (ip == null || ip.length() == 0 || "unknown".equalsIgnoreCase(ip)) {  
-            ip = request.getHeader("HTTP_X_FORWARDED_FOR");  
-        }  
-        if (ip == null || ip.length() == 0 || "unknown".equalsIgnoreCase(ip)) {  
-            ip = request.getRemoteAddr();  
-        }  
-        return ip;  
-    }  
+
+    private static String getClientIpAddr(HttpServletRequest request) {
+        String ip = request.getHeader("X-Forwarded-For");
+        if (ip == null || ip.length() == 0 || "unknown".equalsIgnoreCase(ip)) {
```

Comment:
You should extract `"unknown".equalsIgnoreCase(ip)` to a variable since it's reused in the other `if`